src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/NuGet.PackageManagement.VisualStudio.csproj

```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Microsoft.Build">
-      <HintPath>$(NuGetPackageRoot)microsoft.build\15.1.262-preview5\lib\net46\Microsoft.Build.dll</HintPath>
+      <HintPath>$(PkgMicrosoft_Build)lib\net46\Microsoft.Build.dll</HintPath>
       <Private>False</Private>
     </Reference>
```

Comment:
Looks like this assembly reference could be removed if we change the `PackageReference` to have `ExludeAssets="runtime"` instead of including `None`.